core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2416,9 +2416,9 @@ public String toString() {
           .build();
   public static final PropertyKey USER_FILE_CREATE_TTL_ACTION =
       new Builder(Name.USER_FILE_CREATE_TTL_ACTION)
-          .setDefaultValue("DELETE")
+          .setDefaultValue("FREE")
```

Comment:
Do you think it makes sense for this to be DELETE by default? For `create`, the user is creating the file in both Alluxio and the UFS (undo this with DELETE), while for `load`, the user is only creating the file in Alluxio (undo this with FREE). @calvinjia

Reply:
IMO `free` is a safer default since it is non-destructive. The complement for `load` would be `delete, alluxioOnly`, which is maybe another TTL mode we should consider.

Reply:
I see why we would want to use FREE as the default for loading TTLs. DELETE + load can delete any data discovered by Alluxio, even if it was written out of band and isn't aware of Alluxio.

My concern is that users will expect that files created with default TTL will be deleted in both Alluxio and the UFS. That seems like a more natural definition of TTL, and that's how it's worked in Alluxio for the last two years. Have you seen instances where users create files with TTL, then are dismayed when the data is deleted from UFS?

Reply:
Use **DELETE** as default for creation. And set OutStreamOptions TTL action "DELETE" by default as well, because i think it is related to writing. WDYT.

Reply:
We should try to maintain backwards compatibility as much as possible so that upgrading to 1.8 doesn't break things for users who have been using TTL in 1.7 and earlier. Previously we didn't have TTLs on load, so we can choose either FREE or DELETE without breaking anything. But for the options that previously included TTL already, we need to stick with DELETE.

I think there's a separate issue where any filesystem operation, including `createFile`, could trigger files to be loaded from the UFS. In this case, the loaded files should use `USER_FILE_LOAD_TTL`, while the created file should use `USER_FILE_CREATE_TTL`. But there's only one TTL in `CommonOptions`. We can handle that in a separate PR from this one.

Reply:
So, i should revert all FREE to DELETE except the TTLs for load?